resreporter/mem_linux.go

```diff
@@ -0,0 +1,70 @@
+package resreporter
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+)
+
+const meminfo = "/proc/meminfo"
+
+var Memory = memory{}
+
+type memory struct{}
+
+func (memory) Used() (used uint64, total uint64) {
+	fd, err := os.Open(meminfo)
+	if err != nil {
+		//XXX Should this use metafora's logger somehow?
+		log.Printf("Error reading free memory via "+meminfo+": %v", err)
```

Comment:
From IRL discussion: we couldn't really think of anything compelling for the balancer to do with errors returned by the reporter, so I'm going to leave this be for v1 unless someone comes up with a great idea before merge time.